codecs/wp2/dec/README.md

```diff
@@ -0,0 +1,17 @@
+# WebP2 decoder
+
+The source for the library is not open-source _yet_. The second it is we will publicize the build steps.
```

Comment:
I'm assuming this can be removed / updated, since we're including a link to the source in the Makefile now.